src/codegen/templates/function_template.py.mako

```diff
@@ -52,12 +54,17 @@
 ## Script function body.
 ################################################################################
 \
-## Script default values for parameters that are lists, since default values in Python functions should not be mutable.
+## Script default values for parameters that are lists or file path,
+## since default values in Python functions should not be mutable.
     %for input_param in func.parameters:
         %if input_param.is_list:
         if ${input_param.parameter_name} is None:
             ${input_param.parameter_name} = ${get_list_default_value(func, input_param)}
 
+        %elif is_path_type(input_param):
+        if not ${input_param.parameter_name}.name:
+            ${input_param.parameter_name} = "" # type: ignore
```

Comment:
I think you should rewrite this to not need `# type: ignore`. For example, if the parameter type is `Union[str, pathlib.PurePath, None]`, then you can do this without `# type: ignore`:

```
if parameter is None:
    parameter = ""
```

When you **do** need `# type: ignore`, use `# type: ignore[category]` to document what you are ignoring.

Reply:
`# type: ignore` is removed in new commit.